workspace/notebook/py_unit_tests_functions.json

```diff
@@ -366,25 +366,18 @@
 			},
 			{
 				"cell_type": "code",
-				"metadata": {
-					"jupyter": {
-						"source_hidden": false,
-						"outputs_hidden": false
-					},
-					"nteract": {
-						"transient": {
-							"deleting": false
-						}
-					}
-				},
 				"source": [
 					"def differentiate_std_and_hrm(std_table_full: str, hrm_table_full: str, data_model_columns: list):\n",
-					"    # filter standardised df with non-null message_type and exclude 'Delete' message_type since it doesn't add a new row in hrm\n",
+					"    # Load the standardised DataFrame\n",
 					"    std_df: DataFrame = spark.table(std_table_full)\n",
+					"\n",
+					"    # Filter to include only specific message types\n",
+					"    allowed_message_types = ['Create', 'update', 'Update', 'Publish', 'Unpublish', 'Delete']\n",
```

Comment:
If there is are `update` and `Update` entries, could that also be true for the other message types? Might be better if the col is converted to lower case before comparison?